backend/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "Anonymous Feedback Tool Backend",
+  "main": "index.js",
+  "dependencies": {
+    "apollo-datasource-rest": "3.7.0",
+    "apollo-server-express": "3.11.1",
+    "cookie-parser": "1.4.6",
+    "cors": "2.8.5",
+    "debug": "4.3.4",
+    "dotenv": "10.0.0",
+    "ejs": "3.1.8",
+    "express": "4.18.2",
+    "express-jwt": "6.1.1",
+    "graphql": "15.5.3",
+    "graphql-import": "1.0.2",
+    "graphql-scalars": "1.20.1",
+    "nodemailer": "6.8.0",
+    "passport": "0.6.0",
+    "passport-http": "0.3.0",
+    "pg": "8.8.0"
+  },
+  "devDependencies": {
+    "babel-eslint": "10.1.0",
+    "eslint": "7.32.0",
+    "eslint-config-airbnb-base": "14.2.1",
+    "eslint-config-prettier": "8.5.0",
+    "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-jsx-a11y": "6.6.1",
+    "eslint-plugin-prettier": "4.2.1",
+    "nodemon": "2.0.20",
+    "npm-check": "5.9.2",
+    "prettier": "2.8.1"
+  },
+  "scripts": {
+    "start": "nodemon -e js,graphql -x node src/index.js",
+    "start:prod": "ENVIRONMENT=prod node src/index.js",
+    "dev": "nodemon -e js,graphql -x ENVIRONMENT=dev node --inspect src/index.js",
+    "win-dev": "SET ENVIRONMENT=dev&& nodemon -e js,graphql -x node --inspect src/index.js",
+    "test": "jest",
+    "upgrade-interactive": "npm-check --update",
+    "lint": "eslint src --ext .js",
+    "lint:fix": "npm run lint -- --fix"
+  },
+  "keywords": [],
+  "contributors": [
+    "Ismail Orabi <ismail.orabi@yale.edu> (https://git.yale.edu/iao4)",
+    "Aliza Knight <aliza.knight@yale.edu> (https://git.yale.edu/ak2746)"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@git.yale.edu:LMS/anonymous-feedback.git"
+  },
```

Comment:
You can remove the `contributors` or update to our public github info.

Update repo link to public github

Reply:
@RennKat Can you make the same change to the backend `package.json`

Reply:
@iorabi Sorry about that - fixed!